src/coreComponents/dataRepository/BufferOps_inline.hpp

```diff
@@ -1735,7 +1736,6 @@ Unpack( buffer_unit_type const * & buffer,
     }
 
     arraySlice1d< localIndex, USD - 1 > varSlice = var[li];
-    array1d< globalIndex > unmappedIndices;
 
```

Comment:
can `unmappedIndices` be filled in the `Unpack` call? Adding a `resize` will preserve the previous behavior.
```suggestion
unmappedIndices.resize(0);
```